vignettes/getting-started.Rmd

```diff
@@ -0,0 +1,156 @@
+---
+title: "Getting Started with rbabylon"
+author: "Seth Green"
+date: "2020-03-20"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{getting-started}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r cleanup, include=FALSE}
+# delete old files
+# if feels like I shouldn't need to do this, but R CMD CHECK fails without it
+if (fs::file_exists("../inst/nonmem/1.yaml")) fs::file_delete("../inst/nonmem/1.yaml")
+if (fs::file_exists("../inst/nonmem/2.yaml")) fs::file_delete("../inst/nonmem/2.yaml")
+if (fs::file_exists("../inst/nonmem/3.yaml")) fs::file_delete("../inst/nonmem/3.yaml")
+if (fs::file_exists("../inst/nonmem/2.ctl")) fs::file_delete("../inst/nonmem/2.ctl")
+if (fs::file_exists("../inst/nonmem/3.ctl")) fs::file_delete("../inst/nonmem/3.ctl")
+if (fs::file_exists("../inst/nonmem/babylon.yaml")) fs::file_delete("../inst/nonmem/babylon.yaml")
+```
+
+# Introduction
+
+`rbabylon` is an R interface for running `babylon` which, according to [its README](https://github.com/metrumresearchgroup/babylon/), "is (will be) a complete solution for managing projects involving modeling and simulation with a number of software solutions used in pharmaceutical sciences." Currently, only NONMEM is supported, so this vignette will only address that. 
+
+This "Getting Started with rbabylon" vignette takes the user through some basic scenarios for modeling with NONMEM using `rbabylon`, introducing you to its basic workflow and functionality. 
+
+# Setup
+
+There is some initial setup to get `rbabylon` configured. First, load the library and make sure you have installed `babylon` (often aliased as `bbi`) and told `rbabylon` where to find it. **POINT TO BBI INSTALLATION INSTRUCTIONS, do these exist?**
+
+```{r bbi_exe_path}
+library(rbabylon)
+options('rbabylon.bbi_exe_path' = '/data/apps/bbi')
```

Comment:
I tried putting this in Rprofile but it didn't work. would be nice if we could put this in the background similar to `pkgr`